### PR TITLE
chore: release notes 1.15

### LINKED ISSUE #9742: Podman Desktop 1.15 Release Notes
### Is your enhancement related to a problem? Please describe

Complete Podman Desktop 1.15 Release Note

### Describe the solution you'd like

Complete Podman Desktop 1.15 Release Note

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by axel7083**
### What issues does this PR fix or reference?

Fixes https://github.com/podman-desktop/podman-desktop/issues/9742

### Tests

Preview https://676019e6e1536c303feb17be--podman-desktop-pr.netlify.app/blog/podman-desktop-release-1.15

### CODE REVIEW

File: website/blog/2024-12-12-release-1.15.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+Get the latest release from the [Downloads](/downloads) section of the website and boost your development journey with Podman Desktop. Additionally, visit the [GitHub repository](https://github.com/containers/podman-desktop) and see how you can help us make Podman Desktop better.
+
+---
+
+## Detailed release changelog
+
+#### Documentations 📚
+
+* docs(website): edited the doc to provide clarity in using the libkrun… by @shipsing in [#10111](https://github.com/podman-desktop/podman-desktop/pull/10111)
+* docs(website): edited the managing objects section by @shipsing in https://github.com/podman-desktop/podman-desktop/pull/9845
```
Comment (gastoner) [inline website/blog/2024-12-12-release-1.15.md]:
```suggestion
* docs(website): edited the managing objects section by @shipsing in [#9845](https://github.com/podman-desktop/podman-desktop/pull/9845)
```

File: website/blog/2024-12-12-release-1.15.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+## Final notes
+
+### Fixed issues
+
+The complete list of issues fixed in this release is available [here](https://github.com/containers/podman-desktop/issues?q=is%3Aclosed+milestone%3A1.15.0).
+
+### Where to download
+
+Get the latest release from the [Downloads](/downloads) section of the website and boost your development journey with Podman Desktop. Additionally, visit the [GitHub repository](https://github.com/containers/podman-desktop) and see how you can help us make Podman Desktop better.
```
Comment (feloy) [inline website/blog/2024-12-12-release-1.15.md]:
```suggestion
Get the latest release from the [Downloads](/downloads) section of the website and boost your development journey with Podman Desktop. Additionally, visit the [GitHub repository](https://github.com/podman-desktop/podman-desktop) and see how you can help us make Podman Desktop better.
```

File: website/blog/2024-12-12-release-1.15.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+* [@sozercan](https://github.com/sozercan) made their first contribution in [#10082](https://github.com/podman-desktop/podman-desktop/pull/10082)
+* [@mhdawson](https://github.com/mhdawson) made their first contribution in [#10178](https://github.com/podman-desktop/podman-desktop/pull/10178)
+
+---
+
+## Final notes
+
+### Fixed issues
+
+The complete list of issues fixed in this release is available [here](https://github.com/containers/podman-desktop/issues?q=is%3Aclosed+milestone%3A1.15.0).
```
Comment (feloy) [inline website/blog/2024-12-12-release-1.15.md]:
```suggestion
The complete list of issues fixed in this release is available [here](https://github.com/podman-desktop/podman-desktop/issues?q=is%3Aclosed+milestone%3A1.15.0).
```

File: website/blog/2024-12-12-release-1.15.md
Comment (shipsing) [inline website/blog/2024-12-12-release-1.15.md:1]: Suggestions:
In the [Feedback Categories](https://675c4b61288e05159b849e05--podman-desktop-pr.netlify.app/blog/podman-desktop-release-1.15#feedback-categories) section, we can present the categories in a bulleted list. A numbered list usually represents the sequential order of performing a task. 
We can also map the category names with the names shown in the UI:
- **Direct your words to the developers**
- **Feature request**
 - **Bug**

Suggestion for the `Enabling the Experimental Task Manager` section:
To try it out, go to **Settings > Preferences > Tasks** and enable the **Manager** option.

Second stmt in the community Thank you section, add a comma after `In this release`.

The rest of the content looks good.

Comment (axel7083) [reply website/blog/2024-12-12-release-1.15.md:1]: Thanks for the valuable suggestions!

**Review by Firewall — APPROVED** [review]

File: website/blog/2024-12-12-release-1.15.md
Comment (shipsing) [reply website/blog/2024-12-12-release-1.15.md:1]: @axel7083, please change **Settings -> Preferences > Tasks** to **Settings -> Preferences > Tasks**
Replace -> with >

We need to map the category names with the UI:
Also, Change **Direct Message to Developers** to **Direct your words to the developers** and Bug report to **Bug**.

**Review by shipsing — APPROVED** [review]
Thanks for making changes, @axel7083

**Review by cdrage — APPROVED** [review]
Amazing work on the release and blog!

This all LGTM. Great work :)

**Pull request merged**